patches/google_apis-google_api_keys.cc.patch

```diff
@@ -9,4 +9,4 @@ index ca4572f5fa484fe334ce7f9f8c1821be4b6ed6e0..7c542a0c157485b36444eabc364d7bc9
 +  void set_api_key_for_testing(const std::string& api_key) { api_key_ = api_key; }
```

Comment:
I don't think this patch is necessary and it seems to need manual updating frequently based on history. @yrliou the original is here https://github.com/brave/brave-core/commit/70def0c486d9945c7be1946b346229156492be9e but I'm not seeing why this is necessary.

Reply:
It's a while back and yea it's very likely there are other ways to do this now, cc @atuchin-m who is now much more familiar with our translate feature for removing this.

Reply:
In fact the method is still used in unit_tests: https://github.com/brave/brave-core/blob/master/chromium_src/components/translate/core/browser/translate_manager_unittest.cc#L173

But `google_apis::HasAPIKeyConfigured()` [has a custom override](https://github.com/brave/brave-core/blob/master/chromium_src/components/translate/core/browser/translate_manager.cc#L14) for `TranslateManager`. IMHO we don't need to test set or unset API key in unit_tests, just the main case that it works by default.

So I'm okay to remove it, but the tests need to be changed.

Reply:
Create a follow up issue https://github.com/brave/brave-browser/issues/27338